datadog/dogshell/service_level_objective.py

```diff
@@ -25,7 +30,6 @@ def setup_parser(cls, subparsers):
         create_parser = verb_parsers.add_parser("create", help="Create a SLO")
         create_parser.add_argument(
             "--type",
-            required=True,
```

Comment:
Are this and the following not required anymore ?

Reply:
it is, but I was getting argument parser errors adding this :(

Reply:
FWIW the API will 400 if it's not provided, I think this is okay